spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -267,8 +267,8 @@
         value: VMware Tools
     context_type: hash
     '
-    expect(filter.to_ruby).to eq("<value type=string>guest_applications.name</value> == \"VMware Tools\"")
-    expect(Condition.subst(filter.to_ruby, data, {})).to eq("\"VMware Tools\" == \"VMware Tools\"")
+    expect(filter.to_ruby).to eq("<value type=string>guest_applications.name</value> == \"vmware tools\"")
```

Comment:
Similarly, this is changing what I understand about Ruby. We know that its `==` operator is not case insensitive, so this expression is now misleading to me